java/src/main/java/vowpalWabbit/learner/VWLearner.java

```diff
@@ -10,4 +11,9 @@
 
     // This tells the implementations that an IOException cannot be thrown.
     @Override void close();
+
+    void saveModel(File filename);
+    
+    // Get the JNI pointer for the learner
+    public long getNativePointer();
```

Comment:
@jon-morra-zefr @JohnLangford @stephendavidmarsh isn't exposing the `getNativePointer()` method in the interface a good bad idea.  Seems like there is little practical use for this and it will likely only be used in nefarious ways.